### ISSUE TITLE
[bug] Error setting up gmail account

### ISSUE THREAD

**Issue by Joly0** [crocodilestick/Calibre-Web-Automated#76]
**Describe the bug**
I have followed the steps in the guide to create a google app with the needed apis and mounted the gmail.json to the cwa docker container. When i change the email account type to gmail account and click on setup gmail account i get the error "could not locate runnable browser"

**To Reproduce**
1. Add gmail.json to the docker container
2. change email account type to gmail account
3. click on setup gmail account

**Expected behavior**
Should work

**Screenshots**
![grafik](https://github.com/user-attachments/assets/116d0734-760d-41fd-968a-57439c31f4d5)

**Configuration(please complete the following information):**
 - OS: Unraid
 - Hardware: x86 server
 - Version: V2.0.1

**Additional context**

**Comment by iroQuai** [comment]
I've got the same error. This is a thing with regular Calibre web aswel! I've tried several things without succes (mainly the steps mentioned in https://github.com/janeczku/calibre-web/issues/2254#issuecomment-2211140123, but also the tips in the wiki: https://github.com/janeczku/calibre-web/wiki/Dependencies-in-Calibre-Web-Linux-and-Windows#optional-features-for-installation-with-pip) 

Don't know if this should be in Calibre web automated issues, or in the original project!
